package/yast2-schema-micro.spec

```diff
@@ -0,0 +1,112 @@
+#
+# spec file for package yast2-schema
+#
+# Copyright (c) 2013 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           yast2-schema-micro
+Version:        4.4.9
+Release:        0
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+Source0:        yast2-schema-%{version}.tar.bz2
+
+Group:	        System/YaST
+License:        GPL-2.0-or-later
+
+Url:            https://github.com/yast/yast-schema
+
+# provide schema, so it is full replacement
+Provides:       yast2-schema
+# both cannot be used as it uses same files
+Conflicts       yast2-schema
+
+# Dependencies needed to build the package
+BuildRequires:  jing
+BuildRequires:  libxml2-tools
+BuildRequires:	trang yast2-devtools
+
+# All packages providing RNG files for AutoYaST
+# in /usr/share/YaST2/schema/autoyast/rng/*.rng
+
+# add 'efi' element to the rules schema
+BuildRequires: autoyast2 >= 4.4.21
+BuildRequires: yast2
+# add_on_products and add_on_others types
+BuildRequires: yast2-add-on >= 4.3.3
+# set 't' element in 'initrd_module' element
+BuildRequires: yast2-bootloader >= 4.3.12
+BuildRequires: yast2-country >= 4.3.0
+BuildRequires: yast2-configuration-management >= 4.3.0
+# add 'iface' element
+BuildRequires: yast2-iscsi-client >= 4.3.3
+BuildRequires: yast2-kdump
+# Added backend to the networking section
+BuildRequires: yast2-network >= 4.4.29
+# registration is available only where suse connect is also available
+%ifnarch s390 %ix86
+# addons: architecture/version is optional
+BuildRequires: yast2-registration >= 4.3.12
+%endif
+# Added lsm 'none' section to security.rnc
+BuildRequires: yast2-security >= 4.4.7
+BuildRequires: yast2-services-manager
+# tag home_btrfs_subvolume
+BuildRequires: yast2-users >= 4.1.11
+
+
+#!BuildIgnore: yast2-build-test
```

Comment:
Could not get rid of all these comments?

Reply:
well, it is not comments, I think it is directives for build service to ignore some dependencies...I just worry that some of them are already obsolete